packages/lit-dev-content/src/icons/check-code-icon.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
So far we've been naming these files with the name of the icon, rather than where they are used. An advantage of that is that when you are looking to see if we have some material icon, you can look in the `icons/` directory to find it. The same icon might be useful in two contexts on the site.